### ISSUE TITLE
Check 18 - DistributedCacheAdministration PowerShell Module

### ISSUE THREAD

**Issue by JvanderMeer-GitHub** [ykuijs/SPPeriodicChecks#5]
If the PSModulePath is managed by a Group Policy and it doesn't contain the path for the DistributedCacheAdministration module the check will fail. To avoid a Group Policy change you can import the module directly. 

```powershell
Import-Module "C:\Program Files\AppFabric 1.1 for Windows Server\PowershellModules\DistributedCacheAdministration"
```

**Comment by ykuijs** [comment]
Will update the script to check if this path exists and if so, load the module from there.

**Comment by ykuijs** [comment]
Added in next version

**Comment by ykuijs** [comment]
Fixed via PR #12
